docs/api/python/getting-started.md

```diff
@@ -43,12 +43,12 @@ Once you have all the necessary dependencies, connect to zkSync Era using the en
 ```python
 from zksync2.module.module_builder import ZkSyncBuilder
 ...
-sdk = ZkSyncBuilder.build("https://testnet.era.zksync.dev")
+sdk = ZkSyncBuilder.build("https://sepolia.era.zksync.dev")
 ```
 
 ## Examples
 
-Full code for all examples is available [here](https://github.com/zksync-sdk/zksync2-python/tree/master/examples). Examples are configured to interact with `zkSync` and `Goerli` test networks.
+Full code for all examples is available [here](https://github.com/zksync-sdk/zksync2-python/tree/master/examples). Examples are configured to interact with `zkSync`, `Sepolia` or `Goerli` test networks.
```

Comment:
Did we configure the examples for Sepolia?

Reply:
https://github.com/zksync-sdk/zksync2-python/commit/1918fffc41d6e6a3f35b89a3001186fe7224df04

Reply:
```suggestion
Full code for all examples is available [here](https://github.com/zksync-sdk/zksync2-python/tree/master/examples). Examples are configured to interact with `zkSync` and `Sepolia` test networks.
```